test/end2end_test.go

```diff
@@ -4192,8 +4192,10 @@ func TestServerCredsDispatch(t *testing.T) {
 	}
 	defer cc.Close()
 
+	rawConn := cred.getRawConn()
+	time.Sleep(100 * time.Millisecond)
```

Comment:
Why 100ms?  Sleeps make me nervous.

Reply:
In the normal cases, when `handshake` returns an error, `rawConn` gets closed.
In this test, `handshake` returns a special error, which tells gRPC to not close the connection.
This is to sleep and wait for gRPC to see the error, and then test if the connection was closed.
`100ms` is picked randomly...

Reply:
:disappointed: OK.  Maybe a comment here that says "give grpc a chance to see the error and potentially close the connection"?